FAQ.md

```diff
@@ -0,0 +1,46 @@
+# FAQ
+
+## ADAL vs MSAL
+
+MSAL.js integrates with the Azure AD v2.0 endpoint, whereas ADAL.js integrates with the Azure AD v1.0 endpoint. The v1.0 endpoint supports work accounts, but not personal accounts. The v2.0 endpoint is the unification of Microsoft personal accounts and work accounts into a single authentication system. Additionally, with MSAL.js you can also get authentications for Azure AD B2C.
```

Comment:
Do we want to explicitly state that `MSAL` also supports the v1 endpoint? Or is there already a previous assumption that's made where the v2 endpoint is backwards compatible with the v1 endpoint?

Reply:
It supports v1 tokens, but still uses the v2 endpoint, is my understand.

Reply:
I believe that is correct as well.

Reply:
https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-compare-msal-js-and-adal-js#choosing-between-adaljs-and-msaljs

Reply:
Can we just link to this instead?

Reply:
Done. Linked to the relevant documentation and added one more section